connector.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2020 Snowflake Computing Inc. All right reserved.
+
+package gosnowflake
+
+import (
+	"context"
+	"database/sql/driver"
+)
+
+// Driver is the interface for a Snowflake driver
+type Driver interface {
```

Comment:
Don't think it is a good idea while importing database/sql/driver we create another 'Driver' interface here. Could you just use a wrapper to SnowflakeDriver outside of gosnowflake package?

Reply:
> Don't think it is a good idea while importing database/sql/driver we create another 'Driver' interface here.

Why is that a problem?

Reply:
made this Driver private